### ISSUE TITLE
Recovery process is not starting even though extension is installed

### ISSUE THREAD

**Issue by adrian-delgado** [passbolt/passbolt_browser_extension#121]
## Problem recovering account after firefox update
* Passbolt Version: 2.13.5
* Platform and Target: Manjaro Linux & MacOS
-- Browser: Mozilla Firefox 85.0.1 
-- Web server: -
-- Passbolt Firefox Extension version: 3.0.5

### What you did
I was recovering my account, and received a recovery link.

### What happened
When opening the recovery link (received from the email), it says that the passbolt extension needs to be installed, but the extension is already installed.

Browser console logs:
```
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). injectGlobalHook.js:808:49
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). onloadwff.js:71:791213
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). fingerprint-protection.js:343:25
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. <some-uid>
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). 2 fingerprint-protection.js:343:25
uncaught exception: Object recover.js:401:2675
```

### What you expected to happen
The recovery page should recognize that I have the extension installed.

### Further info
I tried this from another machine (macbook) and it worked, but today I upgraded firefox on my macbook as well and it stopped working. both my machines now have firefox 85.0.1 and it's not working on either of them.

**Comment by adrian-delgado** [comment]
I _just_ tried it again and it works now. at least one other person has had the same issue (same browser, same version) and it's working for them as well now

**Comment by stripthis** [comment]
hi @adrian-delgado,

Do you have any other plugins installed by any chance running along side of passbolt?
Is the browser console log from the page or for the extension? (e.g. did you do right click > inspect element > console or did you go to about:devtools-toolbox?id=passbolt%40passbolt.com&type=extension)

**Comment by adrian-delgado** [comment]
hey @stripthis, I do have other plugins. let me know if it makes sense to give the list of plugins.

The console log is from the page (opened developer tools).

If it helps, before having this issue, I kept getting an error saying there was an 'internal server error', and to 'contact my administrator' (I did not check the console then); but others were able to log into passbolt just fine, myself included when I checked from my laptop (before upgrading firefox). So I assumed the issue was with either firefox, or the extension. 

I was also checking the logs on the server, and there were no errors server-side while I was getting the error on the client.

Hope this helps
